src/ec/packages/ec-component-hero-banner/hero-banner.story.js

```diff
@@ -15,105 +16,97 @@ import heroBanner from './ecl-hero-banner.html.twig';
 import notes from './README.md';
 
 const PrepareBanner = data => {
+  getExtraKnobs(data);
```

Comment:
So, it turns out that we have, actually, control on the order of the knobs, it's just the way you place them in the code.
That's why you saw this getExtraKnobs call always at the end of the processing function, because these are normally the last ones we want to show ;)